plugin/src/main/java/io/jenkins/plugins/casc/Attribute.java

```diff
@@ -225,18 +234,28 @@ public CNode describe(Owner instance, ConfigurationContext context) throws Confi
                     ": No configurator found for type " + type);
         }
         try {
-            Object o = getValue(instance);
-            if (o == null) {
-                return null;
+            Object o;
+            if(isJsonSchema) {
+                 o = getType();
+                if (o == null) {
+                    return null;
+                }
+            } else {
+                 o = getValue(instance);
+                if (o == null) {
+                    return null;
+                }
             }
 
             // In Export we sensitive only those values which do not get rendered as secrets
             boolean shouldBeMasked = isSecret(instance);
             if (multiple) {
                 Sequence seq = new Sequence();
                 if (o.getClass().isArray()) o = Arrays.asList((Object[]) o);
-                for (Object value : (Iterable) o) {
-                    seq.add(_describe(c, context, value, shouldBeMasked));
+                if (o instanceof Iterable) {
```

Comment:
From debugging in my experience most of the values just end up non Iterable :sweat:

Reply:
Should we add a system log warning for that? Maybe on the FINE level

Reply:
Yeah I guess so.